code/src/index.css

```diff
@@ -11,3 +11,114 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 10vh;
+  text-align: center;
+}
+
+.input-fields {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 15px;
+}
+
+.input-fields input {
+  border: 2px solid grey;
+  border-radius: 8px;
+  font-size: 20px;
+  width: 40vw;
+  height: 3vh;
+}
+  
+.input-fields select {
+    border: 2px solid grey;
+    border-radius: 8px;
+    font-size: 15px;
+    width: 40vw;
+    height: 5vh;
+    text-align: center;
+}
+
+label {
+  margin: 10px;
+}
+
+.form-response-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  margin-top: 10vh;
+}
+
+form div {
+  text-align: center;
+}
+
+.submit-button {
+  margin-top: 10vh;
+  font-size: 20px;
+  border-radius: 10px;
+  border: none;
+  padding: 8px;
+  background-color: lightblue;
+}
+
+.submit-button:hover, .next-button:hover  {
+  filter: brightness(90%);
+}
+
+.next-button {
+  margin-top: 7vh;
+  font-size: 20px;
+  border-radius: 10px;
+  border: none;
+  padding: 8px;
+  background-color: lightgreen;
+}
+
+.next-and-submit-button-div {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
I noticed that the submit button was present throughout the whole survey but it is only able to be clicked in the end of the survey. Is there a specific reason why it is shown in all your components? 

I noticed that you hid the next button on the last page and just showed the submit button. I wonder if maybe you would want to do the same with the submit button, hide it until the user needs to submit the form?